frontend-build-pipeline.yml

```diff
@@ -16,30 +29,30 @@ steps:
 
 - script: |
     npm ci
-    npx ng build --prod
+    npx ng build
   displayName: 'npm Build'
 
 # skipped failing unit tests
 # https://jasmine.github.io/tutorials/your_first_suite#section-Pending_Specs
```

Comment:
that comment would really fit more in the PR description than here, it doesn't really relate to the pipeline itself.

Reply:
Moved to PR description.